login.php

```diff
@@ -0,0 +1,31 @@
+<?php include('includes/header.php');?>
+<div id="page-container">
+
+    <div id="content-wrap">
+
+        <form action="" method="POST">
+
+            <div class="container">
```

Comment:
Move the .container div outside of the <form> and change it to a `<main>` element for semantic reasons